Sources/TokamakDOM/Shapes/Path.swift

```diff
@@ -30,26 +30,30 @@ extension Path: ViewDeferredToRenderer {
       "stroke-width": "\(strokeStyle.lineWidth)",
     ]
     let uniqueKeys = { (first: String, _: String) in first }
+    let width: String? = sizing == .flexible ? "100%" : nil
+    let height: String? = sizing == .flexible ? "100%" : nil
+    let centerX: String? = sizing == .flexible ? "50%" : nil
+    let centerY: String? = sizing == .flexible ? "50%" : nil
```

Comment:
To clarify what these are for, maybe?

```suggestion
    let flexibleWidth: String? = sizing == .flexible ? "100%" : nil
    let flexibleHeight: String? = sizing == .flexible ? "100%" : nil
    let flexibleCenterX: String? = sizing == .flexible ? "50%" : nil
    let flexibleCenterY: String? = sizing == .flexible ? "50%" : nil
```